tests/trestle/core/commands/author/docs_test.py

```diff
@@ -21,11 +21,17 @@
 
 from _pytest.monkeypatch import MonkeyPatch
 
+import frontmatter
+
 import pytest
 
-import trestle.cli
+from tests.test_utils import execute_command_and_assert
+
+import trestle
 from trestle.core.commands.author.consts import START_TEMPLATE_VERSION
 
+import yaml
+
```

Comment:
I thought we were using ruamel.yaml  I think this needs to change here and in markdown_api.py